examples/object_model/ellipses.m

```diff
@@ -0,0 +1,90 @@
+clear all
```

Comment:
as this example introduces this new feature, commeting every line of the new options would be helpful.
currently requires lot of trail and error to figure out what is what  - please make the user's life easy.

also ma27 has a lot of trouble with this example on my machine in the first homotopy iteration. dual inf goes to 1e14. later iterations are easy. 
is the problem well scaled?

Reply:
Interesting... ma27 converges pretty well on my machine (though in the initial phase dual infeasibility does go to 1e8 but that is not uncommon in my experience). Will add comments.